JeffreyQ/UCLA-CS-130
# Rename access tokens references to JWT references, request to create …

…new user on fb auth login, and store JSON Web Token instead of fb user_access_token.

It no longer makes sense to store the user_access_token provided by the Facebook Graph API as we only use it once to retrieve a JWT from the backend. 

The JWT serves as our primary form of identification as we interface with our backend app, so we store it in AsyncStorage and our redux store when we first authenticate and login to Polly.


TESTING 

1. Clear AsyncStorage
2. If the user associated with the fb user_access_token has an entry inside of the Polly Database then delete that user, and save.
3. Restart metro bundler.
4. Spin up backend with `make build`
5. Check to see all users inside of the user table.
6. Authenticate with Facebook and login to the Polly app.
7. Verify that a new user was added to the database.
